.taprc

```diff
@@ -0,0 +1 @@
+check-coverage: false
```

Comment:
Disabling 100% coverage checking?

Reply:
please restore this

Reply:
As mentioned on https://github.com/pinojs/pino-tee/pull/94#issuecomment-1109080621, this is keeping the old behavior (on tap@14 100% coverage enforcement is disabled by default). The nyc bug on windows will prevent us to enable it since it would break our CI.